src/Sylius/Behat/Context/Transform/LexicalContext.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Context\Transform;
+
+use Behat\Behat\Context\Context;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class LexicalContext implements Context
+{
+    /**
+     * @Transform /^"(?:€|£|\$)([^"]+)"$/
+     */
+    public function getPriceFromString($price)
+    {
+        return (int) round(($price * 100), 2);
+    }
+
+    /**
+     * @Transform /^"([^"]+)%"$/
```

Comment:
Won't it be better to use here regex which will match only floats? `([0-9]*\.)?[0-9]+`